src/test/ensemble_config_change_stress.cpp

```diff
@@ -1103,6 +1120,23 @@ class StressPipelineConfigChanges : public TestWithTempDir {
         createConfigFileWithContent(ovmsConfig, configFilePath);
         SPDLOG_INFO("{} end", __FUNCTION__);
     }
+    void checkMetricGreaterThan(const std::string& metricName, double value) {
+        std::string metricOutput = manager.getMetricRegistry()->collect();
+        ASSERT_THAT(metricOutput, ::testing::HasSubstr(metricName + std::string{"{name=\"dummy\",version=\"1\"} "})) << "cannot find dummys " << metricName << " metric";
+        std::regex rgx(std::string{".*"} + metricName + std::string{"\\{name=\"dummy\",version=\"1\"\\} (.*)\n.*"});
+        std::smatch match;
+        ASSERT_TRUE(std::regex_search(metricOutput, match, rgx)) << "cannot find dummys " << metricName << " metric";
+        auto actualVal = ovms::stoi64(match[1]);
+        // TODO: Probably try to run it multiple times in case of 0 to minimize chances of sporadic
```

Comment:
In which scenario do we expect sporadic? We should not leave TODO's right now.

Reply:
Removed todo